lib/reporters/console.js

```diff
@@ -341,6 +341,14 @@ internals.color = function (name, code, enabled) {
         };
     }
 
+    if (enabled && Array.isArray(code)) {
+        const color = '\u001b[' + code[0] + ';' + code[1] + 'm';
```

Comment:
doesn't look safe, you will want to check the length... might also want to do this condition first and else if enabled do the one on line 336